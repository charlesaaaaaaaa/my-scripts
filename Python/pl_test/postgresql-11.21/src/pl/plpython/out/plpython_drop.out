error:  could not connect to server: Connection refused
	Is the server running on host "192.168.0.136" and accepting
	TCP/IP connections on port 58881?
