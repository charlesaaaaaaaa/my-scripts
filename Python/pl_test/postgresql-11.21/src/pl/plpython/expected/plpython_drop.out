--
-- For paranoia's sake, don't leave an untrusted language sitting around
--
SET client_min_messages = WARNING;
SET
DROP EXTENSION plpython3u CASCADE;
DROP EXTENSION
DROP EXTENSION IF EXISTS plpython2u CASCADE;
DROP EXTENSION
